airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -80,6 +88,27 @@ public class DefaultReplicationWorker implements ReplicationWorker {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultReplicationWorker.class);
 
+  private static final Configs CONFIGS = new EnvConfigs();
```

Comment:
Nit:  This will need to change to work in airbyte-workers, as at some point the `EnvConfigs` will be removed in favor of injection via Micronaut for configuration values.  Instead of making these methods static, it would make more sense to turn this class into a singleton with the required configuration passed to it via constructor.